src/Struct/MailingTemplate/MailingTemplatesSources.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Scn\EvalancheSoapStruct\Struct\MailingTemplate;
+
+/**
+ * Contains the source code of the mailing views
+ */
+class MailingTemplatesSources implements MailingTemplatesSourcesInterface
+{
+
+    /** @var string */
+    private $template_email;
```

Comment:
according to the other Classes, Variables should be lower- cc maybe u can use php-cs-fixer to change that